svelte/src/lib/UplotSvelte.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+    import 'uplot/dist/uPlot.min.css';
+    import type uPlot from 'uplot';
+    import {onDestroy, onMount} from "svelte";
+    // import {optionsUpdateState} from "uplot-wrappers-common";
```

Comment:
The project uses yarn workspaces. You need to add a new entry to the root package.json file in the workspaces section to avoid copying the library

Reply:
Apologies, I was not familiar with yarn and its workspaces feature. This now works and the problem isn't present. 

As a side question: would you say that yarn is more preferable to npm generally? Thanks!

Reply:
Can't say for everyone, but the recent project I worked with all using yarn.
The reason is that yarn provides more features like resolutions, workspaces etc. Having said that it seems like npm started to catch up with yarn a bit in terms of functionality, so it may not be an issue anymore.